docs/tutorials/fundamentals/viewer.md

```diff
@@ -12,61 +12,46 @@ kernelspec:
-launch the viewer. For help with installation see our
-[installation](./installation) tutorial. For help launching the
-viewer see our [getting started](./getting_started) tutorial.
-
-This tutorial will teach you about the **napari** viewer, including how to use
-its graphical user interface (GUI) and how the data within it is organized. At
-the end of the tutorial you should understand the both the layout of the viewer
-on the screen and the data inside of it.
+This tutorial assumes you have already installed **napari** and know how to launch the viewer. For help with installation see our [installation](./installation) tutorial. For help launching the viewer see our [getting started](./getting_started) tutorial.
 
-+++
+This tutorial will teach you about the **napari** viewer, including how to use its graphical user interface (GUI) and how the data within it is organized. At the end of the tutorial, you should understand both the layout of the viewer on the screen and the data inside of it.  
+</br>
 
 ## Launching the viewer
 
-As discussed in the [getting started](./getting_started) tutorial, the napari viewer
-can be launched from the command-line, a python script, an IPython console, or a
-Jupyter notebook. All four methods launch the same viewer, and anything related
-to the interacting with the viewer on the screen applies equally to all of them.
-We will use the python script syntax for any example code which you can copy and
-paste into a script/console/notebook and run.
+As discussed in the [getting started](./getting_started) tutorial, the napari viewer can be launched from the command-line, a python script, an IPython console, or a Jupyter notebook. All four methods launch the same viewer, and anything related to interacting with the viewer on the screen applies equally to all of them. We will use the syntax for running the code inside a jupyter notebook with each code block below pasted into its own cell, but if you'd like to use a python script instead, simply copy and paste the code blocks into scripts with [napari.run()](https://napari.org/stable/api/napari.html#napari.run) as the final line (this starts an event loop which will 
+open an interactive viewer) and run them.  
 
-Let's get stated by launching a viewer with a simple 2D image.
+**Note:** There is also an IPython console available in napari which you can open with the  
+        iPython console button (far left viewer button) or with the menu option **Window** > **console**. You can use this console to programmatically interact with an open viewer using the API methods illustrated in 
+        this tutorial.  
 
-The fastest way to open a viewer with an image on the canvass is
-using {func}`imshow<napari.imshow>`:
+Let's get started by launching a viewer with a simple 2D image.  
 
+The fastest way to open a viewer with an image on the canvas is using {func}`imshow<napari.imshow>`:
 
-```{code-cell} python
+```{code-cell} python  
```

Comment:
Here's the short version: Even though ipython is installed with "python -m pip install -r requirements.txt" when the Build Docs workflow is triggered, something is wrong (unfortunately I don't understand why). On my local machine, installing ipython with conda, in the same environment, solves this problem. Now, I'm not sure what the right solution for this is, since all the other requirements are installed using pip inside of conda. I suspect this (mixing pip and conda) is also the source of a few other problems, such as not being able to build the docs with python=3.10, but that can be discussed separately.

Reply:
FYI, it has the proper highlighting for me now when downloading the built artifact.

Reply:
Fantastic!